src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +70,58 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    class PostalCode {
+        public String postalCode;
+
+        public PostalCode(String postalCode) {
+            this.postalCode = postalCode;
+        }
+
+        @Override
+        public String toString() {
+            return postalCode;
+        }
+    }
+
+    class Block {
+        public String blkName;
```

Comment:
Name doesn't follow naming convention. Always try to use understandable names instead of abbreviations so that other developers can understand.